### ISSUE TITLE
Import failed (missing information?)

### ISSUE THREAD

**Issue by DPalmz** [Glagan/SyncDex#58]
I tried importing my list from both MAL and Ani but neither of them imported correctly.

```
{"logs":[{"d":1633387006517,"msg":"Installation version 0.3.4"},{"d":1633387164531,"msg":"Mochi error: code 0 body undefined"},{"d":1633387164692,"msg":"Mochi error: code 0 body undefined"},{"d":1633387164892,"msg":"Error on request [https://api.mangadex.org/v2/user/me]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"},{"d":1633387189654,"msg":"Mochi error: code 0 body undefined"},{"d":1633387190160,"msg":"Mochi error: code 0 body undefined"},{"d":1633388454290,"msg":"Mochi error: code 0 body undefined"},{"d":1633388454450,"msg":"Mochi error: code 0 body undefined"},{"d":1633388454678,"msg":"Error on request [https://api.mangadex.org/v2/user/me]: SyntaxError: Unexpected token < in JSON at position 0>> SyntaxError: Unexpected token < in JSON at position 0"}],"options":{"acceptLowScore":false,"biggerHistory":true,"chapterStatus":false,"chaptersSaved":400,"checkOnStartup":false,"checkOnStartupCooldown":30,"checkOnStartupMainOnly":true,"colors":{"higherChapter":"transparent","highlights":["rgba(82, 190, 90, 0.6)","rgba(107, 177, 95, 0.6)","rgba(55, 168, 61, 0.6)"],"lowerChapter":"rgba(180, 102, 75, 0.5)","nextChapter":"rgba(104, 115, 251, 0.4)","openedChapter":"rgba(28, 135, 141, 0.4)"},"confirmChapter":true,"displayProgressUpdated":true,"displaySyncStart":false,"errorDuration":4000,"errorNotifications":true,"favoriteLanguage":"all","groupTitlesInLists":true,"hideHigher":false,"hideLast":false,"hideLower":true,"highlight":true,"iconsSilentAfterSync":false,"infoDuration":4000,"linkToServices":true,"logLevel":0,"mdUpdateSyncDex":false,"noReloadStatus":true,"notifications":true,"originalThumbnail":false,"overviewMainOnly":true,"progressInThumbnail":true,"saveOnLastPage":false,"saveOnlyHigher":true,"saveOnlyNext":false,"saveOpenedChapters":true,"separateLanguages":true,"services":["mal","al","mu"],"silentUpdate":false,"subVersion":4,"successDuration":4000,"thumbnail":true,"thumbnailMaxHeight":80,"tokens":{"anilistToken":"set"},"updateMD":true,"updateMDProgress":false,"updateOnlyInList":false,"useMochi":true,"version":0.3}}
```

**Comment by Glagan** [comment]
Sorry but SyncDex is not working for the new version of MangaDex, yet.

Update is in progress but no ETA, I have a working version but it's really barebone and probably full of bugs, maybe I will upload a beta with really limited functionalities "soonish", but without import or export I think, since MangaDex changed how it works.

I fixed your error in the meantime, I broke mochi while changing host recently. But importing now is useless, I don't know if I will import your save on the next version since the architecture changed.
